src/posts/lucky_0.25.cr

```diff
@@ -0,0 +1,370 @@
+class Lucky025Release < BasePost
+  title "Lucky v0.25 is hot off the press, and there's a lot!"
+  slug "lucky-0_25-release"
+
+  def published_at : Time
+    Time.utc(year: 2020, month: 12, day: 19)
+  end
+
+  def summary : String
+    <<-TEXT
+    Lucky v0.25 is out. A ton has changed,
+    and we're full steam ahead to 1.0!
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    [Lucky v0.25](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#changes-in-025)
+    is out now, and we have a ton to go over. Let's get to it!
+
+    ### How to upgrade
+
+    See the [UPGRADE_NOTES](https://github.com/luckyframework/lucky/blob/master/UPGRADE_NOTES.md#upgrading-from-024-to-025).
+
+    Or you can view the changes using [LuckyDiff](https://luckydiff.com/?from=0.24.0&to=0.25.0).
+    This tool helps see what has changed between versions and will be included in upgrade notes from now on.
+
+    ## Here's what's new
+
+    Since the last release we've welcomed two more developer to the core team, [Matthew](https://github.com/matthewmcgarvey) and [Stephen](https://github.com/stephendolan).
+    These two have both been huge contributors in the commuinity, and since bringing them on, the movement towards 1.0 has more than doubled in speed!
+
+    Over the course of the last few months, we've made a ton of changes for this release, so let's break down a couple of the more notible changes:
+
+    ### `Avram::Operation` got a facelift
+
+    When you need to handle some logic that may not be tied directly to a single model, you can create an `Avram::Operation` to handle this. You've probably seen
+    examples in your app like the `RequestPasswordReset` and `SignInUser` that come with an Auth app.
+
+    Prior to this update, it was sort of "wild west", and completely up to you on how to implement these. For consistency sake, we recommended defining a `submit`
+    method, and then returning `yield self, value`. These operations were also limited with no way to use callbacks, file attributes, or define errors not tied to
+    an attribute.
+
+    In this release, we've created a whole new interface!
```

Comment:
Yeah, you have to define `run` now. Before you could have called the method whatever you wanted because it was up to you to use whatever you defined.